plugins/module_utils/client.py

```diff
@@ -63,10 +64,20 @@ def __init__(
         self.host = host
         self.username = username
         self.password = password
+        self.timeout = timeout
 
         self._auth_header = None
         self._client = Request()
 
+    @classmethod
+    def get_client(cls, cluster_instance: dict):
```

Comment:
Added this `classmethod` in order to avoid writing the same code in every module.
Plus it generates an instance of class Client.

Reply:
Nice, I aggree